src/sid/statistics.py

```diff
@@ -76,12 +82,14 @@ def calculate_r_zero(df: pd.DataFrame, window_length: int = 7) -> pd.Series:
     """
     r_effective = calculate_r_effective(df=df, window_length=window_length)
 
+    df["immune"] = df["immunity"] > threshold
```

Comment:
just call this as `not_susceptible` so you don't add a column to states and change it inplace.